docs/pages/custom-builds/schema.mdx

```diff
@@ -740,9 +740,10 @@ You can replace the `eas/build` command call by using these steps in your YAML c
 
 All-in-one function that installs Maestro, prepares a testing environment (Android Emulator or iOS Simulator), and tests the app.
 
-| Input       | Description                                                                                                                                                 |
-| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run. |
+| Input       | Description                                                                                                                                                                                                                   |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run.                                                                   |
+| `app_path`  | **string** Path (or regex pattern) to the simulator/emulator app that should be tested. If not provided defaults to `ios/build/Build/Products/*simulator/*.app` for iOS and `android/app/build/outputs/**/*.apk` for Android. |
```

Comment:
```suggestion
| `app_path`  | `string` Path (or regex pattern) to the emulator/simulator app that should be tested. If not provided defaults **android/app/build/outputs/**/*.apk** for Android and to **ios/build/Build/Products/*simulator/*.app** for iOS. |
```

Android before iOS + formatting nits. 

Would be nice to move the type into separate column in the future, but this improvement is definitely out of this PR scope.

Reply:
I'll follow up to create a `type` separate column, thanks for the feedback @Simek!

We should probably keep `string` in bold text unless we decide to change it for both fields in this table (which might be out of scope of the PR).

Reply:
>Would be nice to move the type into separate column in the future, but this improvement is definitely out of this PR scope.

Yeah, good idea